server/jetstream_cluster_test.go

```diff
@@ -1088,15 +1089,16 @@ func TestJetStreamClusterStreamInfoList(t *testing.T) {
 	sendBatch("baz", 33)
 
 	// Now get the stream list info.
-	sl := js.NewStreamLister()
-	if !sl.Next() {
-		t.Fatalf("Unexpected error: %v", sl.Err())
+	ctx, cancel := context.WithTimeout(context.Background(), 3*time.Second)
+	var infos []*nats.StreamInfo
+	for info := range js.StreamsInfo(ctx) {
```

Comment:
just passing `nil` is fine no? to use the default timeout from the JS context

```suggestion
	for info := range js.StreamsInfo(nil) {
```